### DISCUSSION TITLE
On the production road 🛣

### DISCUSSION THREAD

**Discussion by Yago** [webmardi/webmardi.ch#36] [Announcements]
Hi @CamilleLetang, @WengerK, @gperriard  and @nlsnmr ! 🤗

Here are some status on the website; I almost finish coding the base and now we need some content. As I propose, the events are managed on a Notion database. You can create one or more accounts and I will give you access. Simply **send me the emails that you are using for your Notion account(s)**.

![CleanShot 2021-10-18 at 14 50 24](https://user-images.githubusercontent.com/2025924/137778661-c4972acb-fa02-45ec-8683-27125fbe8b5c.png)

For the moment and until your Infomaniak account have a hosting plan enabled, I upload the website on my Infomaniak account; this way, you can already take a look. There is currently no automatic rebuild, so don't be scared if the events are not updated after a Notion edit 😉

**👉   Here is the [preview](https://4d7d6aoaeg.preview.infomaniak.website/)**

So far, we already have pretty decent notes on Lighthouse and Website Carbon 😄  It's still a WIP, so I can hopefully do better 😉
 
![CleanShot 2021-10-18 at 19 20 32](https://user-images.githubusercontent.com/2025924/137779180-01aec55a-1d81-4a2f-82c8-d7a6b6b4c5c4.png)

![CleanShot 2021-10-18 at 19 20 18](https://user-images.githubusercontent.com/2025924/137779200-ccaa1e45-22e0-4896-ba75-704f4029e432.png)

![CleanShot 2021-10-18 at 19 15 04](https://user-images.githubusercontent.com/2025924/137779212-abfc4bf8-2ed7-44e6-b883-212ebee38602.png)

The next steps are:
- The content on Notion
- Infomaniak activation and setup 
- The Github action to rebuild the website on-demand
- Some optimisation

**Comment by gperriard** [comment]
Hello @Yago ,

Thank you for what you did so far.
Will it be possible to adapt the sponsors, if necessary?

You can use the following accounts for the access:
* geoffroy@webmardi.ch
* kevin@webmardi.ch
* nelson@webmardi.ch
* justine@webmardi.ch
* daniela@webmardi.ch
* camille@webmardi.ch

Regarding infomaniak, I'll let @WengerK answer as I don't have access to it.

**Reply by Yago** [reply]
Hi @gperriard! Sure, we can adapt anything, anytime; it's the purpose of this website 😉

About Notion, we have only 5 guests (accesses) freely included. If you can reduce the list to 5 peoples, it will be perfect.

**Reply by WengerK** [reply]
We can start with a more more "global" access with `contact@webmardi.ch`.

**Comment by Yago** [comment]
After @WengerK gave me access to the Infomaniak hosting, I set up the deploy Github action. So the website is almost ready on https://d36s9anuji.preview.infomaniak.website/ (the preview URL before the DNS configuration for webmardi.ch). I still struggle to create a custom deploy trigger, so for now, the website should rebuild itself 3 times per days around 7am, 12am and 6pm if I understand the Github cron correctly 😉

It remains maybe a bit of content + links review. Also the newsletter form is not finished, but could already work. You need to check on your Mailchimp admin and tell me if not so.

I'll be on holiday for a small week, but on my return, we can make the publication happen as soon as you're ready. If you want to change some text, everything is under the [`/src/locales/en.json`](https://github.com/webmardi/webmardi.ch/blob/next-revamp/src/locales/en.json) 😉

**Reply by WengerK** [reply]
Only things I saw for now:

- We may already add the `hosted by Infomaniak` in footer
- Can we already open a PR where the logo of antistatique is replaced by jolicode (new Gold sponsor) ?

Otherwise ... THAT JUST AWESOME THAAAANKS

**Comment by Yago** [comment]
@WengerK So I just update the sponsors and footer. It remains the newsletter form to complete; **I'll need access to your Mailchimp or the code to use to connect the form**.

Otherwise, if your are OK with the content and links, we can publish this new version 🤗

**Reply by gperriard** [reply]
Hello @Yago ,

Thank you for your changes. I had a look at the code and I'd like to propose small improvements.
I made a PR here https://github.com/webmardi/webmardi.ch/pull/37. I just did the change on the file but I haven't run the whole process though so it would be nice if you could have a look at it :)

Regarding mailchimp I'll let @WengerK handle that.

Regards,
Geoffroy

**Comment by WengerK** [comment]
I'll go to you by dm with the mailchimp stuff.

My only blockers is the release process. Does it release when we changes something ? If not, how can we force a rebuild ?
Otherwise, how can we improve this process ?

**Reply by Yago** [reply]
Github actions is kind of an ass if you want to use and external API trigger; I try, doesn't seems to work. I also set a [cron](https://github.com/webmardi/webmardi.ch/blob/next-revamp/.github/workflows/ci.yml#L5), but also don't “work”. My guess, it's not the default branch or something. So I would like to promote the `next-revamp` branch to `main` and archive `master` to check if it works better.

Otherwise, I will move to another free CI that can handle API trigger. This way you will be hable to trigger a rebuil by click on a Notion button 😉

**Comment by WengerK** [comment]
The new preview link -> https://d36s9anuji.preview.infomaniak.website/
